src/transformers/pipelines/automatic_speech_recognition.py

```diff
@@ -73,13 +73,6 @@ def chunk_iter(inputs, feature_extractor, chunk_len, stride_left, stride_right,
 
         chunk_len = chunk.shape[0]
         stride = (chunk_len, _stride_left, _stride_right)
-        if "input_features" in processed:
-            processed_len = processed["input_features"].shape[-1]
-        elif "input_values" in processed:
-            processed_len = processed["input_values"].shape[-1]
```

Comment:
In most cases, this operation does nothing. 

Prior feature extractors of CTC models output the raw audio waveform, so before W2V2-BERT,  `processed_len` always equaled `chunk_len`.

The operation introduces errors for feature extractors that are using spectrograms-related computations because rescaling is done twice, as explained in the following comment.